src/features/payment/components/PaymentChoice/PaymentMethod.tsx

```diff
@@ -37,21 +37,18 @@ function ImageComponent(method: PaymentInstrumentsType) {
   const [image, setImage] = React.useState<"main" | "alt">("main");
   const onError = React.useCallback(() => setImage("alt"), []);
   const imgSize = { width: "23px", height: "23px" };
-
   const paymentMethodConfig =
     PaymentMethodRoutes[method.paymentTypeCode as PaymentCodeType];
 
   const iconDefault = <DefaultIcon method={method} />;
-
-  if (!paymentMethodConfig?.asset || image !== "main") {
```

Comment:
Dropping this the `image` and `setImage` aren't more useful?

Reply:
I've dropped the commit making this change, it should be fine now